trade-finance-manager-ui/server/routes/auth/configs/index.ts

```diff
@@ -0,0 +1,4 @@
+import { isTfmSsoFeatureFlagEnabled } from '@ukef/dtfs2-common';
+import { getUnauthenticatedAuthSsoRouter } from './unauthenticated-auth-sso';
+
+export const getUnauthenticatedAuthRouter = () => (isTfmSsoFeatureFlagEnabled() ? getUnauthenticatedAuthSsoRouter() : undefined);
```

Comment:
Would this work?

```ts
export const getUnauthenticatedAuthRouter = () => (isTfmSsoFeatureFlagEnabled() && getUnauthenticatedAuthSsoRouter());
```

Reply:
technically, however, will have a rather ugly return type of `false | Router` which feels odd (hence the `undefined`)